utils/execute_command.go

```diff
@@ -47,3 +49,44 @@ func ExecuteCmd(cmd *exec.Cmd, outputCallback func(string)) (string, error) {
 
 	return result, err
 }
+
+// ExecuteAnalysisCmd executes the cmd and returns through outputCallback line-by-line and retrutning only
+// the part that is a valid JSON string
+func ExecuteAnalysisCmd(cmd *exec.Cmd, outputCallback func(string)) (string, error) {
```

Comment:
This analysis can take time as well (for meetings, etc). Wouldn't piping progress to another pipe be better?